--- /build/kanotix/chroot/etc/init.d/kdm	2011-09-28 03:04:16.794899000 +0200
+++ /etc/init.d/kdm	2011-09-28 04:05:22.977140278 +0200
@@ -129,8 +129,6 @@
       log_action_msg "Killing plymouth..."
       if pidof plymouthd >/dev/null; then
         /bin/plymouth --quit --wait
-        /bin/plymouth --wait
-        sleep 1
       fi
       log_daemon_msg "Starting K Display Manager" "kdm"
       if start-stop-daemon --start --quiet $SSD_ARGS -- $ARG ; then
