--- /lib/live/boot/mountroot.sh.orig	2012-06-28 14:39:38.000000000 +0200
+++ /lib/live/boot/mountroot.sh	2012-08-30 06:15:17.559993109 +0200
@@ -80,6 +80,16 @@
 					break
 				fi
 
+				if [ "$i" -eq 10 ]
+				then
+					msg="live-boot has trouble finding your live-media... try unplugging and plugging it in again"
+					if [ -x /bin/plymouth ] && plymouth --ping; then
+						plymouth message --text="$msg"
+					else
+						echo "$msg" >&2
+					fi
+				fi
+
 				sleep 1
 				i="$(($i + 1))"
 			done
